polaris.shopify.com/public/images/foundations/design/spacing/principles.svg

```diff
@@ -0,0 +1,115 @@
+<svg width="800" height="199" viewBox="0 0 800 199" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M8.31632 29.2598H248.031C252.251 29.2598 255.674 32.6925 255.674 36.9306V190.589C255.674 194.827 252.251 198.26 248.031 198.26H8.31632C4.09713 198.26 0.673828 194.827 0.673828 190.589V36.9306C0.673828 32.6925 4.09713 29.2598 8.31632 29.2598Z" fill="#F6F6F7" stroke="#E1E3E5"/>
+<path d="M43.9161 57.2532H43.761L43.6331 57.341L0.5 86.9601V60.251L44.5453 29.2744H73.9101V198.26H45.1225V57.7532V57.2532H44.6225H43.9161Z" fill="#EBEBEB" stroke="#E1E3E5"/>
```

Comment:
This is a huge SVG. I think a PNG would probably be smaller and better optimized for the site.